<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOne">
       <property name="text">
        <string>One</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtoTwo">
       <property name="text">
        <string>Two</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonThree">
       <property name="text">
        <string>Three</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFour">
       <property name="text">
        <string>Four</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFive">
       <property name="text">
        <string>Five</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
